lib/active_shipping/shipment_packer.rb

```diff
@@ -84,7 +84,7 @@ def determine_fillable_quantity_for_package(item, maximum_weight, package_weight
           # Grab the max amount of this item we can fit into this package
           # Or, if there are fewer than the max for this item, put
           # what is left into this package
-          [(maximum_weight - package_weight) / item_grams, item_quantity].min
+          [(maximum_weight - package_weight).to_i / item_grams, item_quantity].min
```

Comment:
I'd assign `maximum_weight - package_weight` to a variable, so we can have fewer things going on in one line. Also makes it super clear what's happening.

```ruby
available_grams = (maximum_weight - package_weight).to_i
[available_grams / item_grams, item_quantity].min
```